package.json

```diff
@@ -24,6 +24,14 @@
                     "type": "string",
                     "default": "",
                     "description": "Configuration file for the code style check"
+                },
+                "haxecheckstyle.sourceFolders": {
+                    "type": "array",
+                    "default": [
+                        "src",
+                        "Source"
```

Comment:
Looks like these are case-sensitive - had some trouble getting this to work in a `source` folder. Gonna fix that. :)

Reply:
Oh, and it's also having issues with forward vs backward slashes. Windows-only issues I guess.